### ISSUE TITLE
Problem with item socets when checking price

### ISSUE THREAD

**Issue by URB4NO** [Kvan7/Exiled-Exchange-2#207]
### Description

Problem is when the item dont have any socket if u wann check price. Item what dont have any socket and u uncheck this option in program nothing change, program i still looking only for items with sockets. If go to trade site of poe u see the Exile Exchange set the valeu of "Empty Rune Socets" to 0, and showing us only items with sockets. If u delete 0 in filter on page u will see more items w/o any socet slot. Any solution of that or it bug ?

### Screenshot

![From game](https://github.com/user-attachments/assets/f6ddab3b-8e46-42e7-af02-e642ea8ee417)
![screen with nothing on socket](https://github.com/user-attachments/assets/e11ea1b3-7e98-43bf-81ce-cd749fd456a4)
![screen with socket 0](https://github.com/user-attachments/assets/88c4672b-50ab-4039-bd66-910e33ea2064)

**Comment by Kvan7** [comment]
well, that is how i wrote it.

but you are right that is definitely wrong

**Comment by Kvan7** [comment]
fixed for next release, (today again maybe?)

**Comment by URB4NO** [comment]
Thanks, btw. u do great job !

**Comment by Llumino24** [comment]
Hey Kvan7!

I just downloaded the 0.2.0 update, and I wanted to give feedback on the above update for Rune Sockets. The above update appears to make the default behavior consider all items with no rune sockets at all to HAVE full empty rune sockets.

Example: This chest piece has no rune sockets at all. When I use the pricing feature, it automatically says that the item has 2 empty runes sockets, limiting the items I am looking at for pricing. This is also reflected when going to the trade site.

![image](https://github.com/user-attachments/assets/41142112-17d3-43a1-aa2a-62aee4ba43e3)

![image](https://github.com/user-attachments/assets/2b0a6e5f-d23e-4635-adbe-90076b1bf2b2)

I do have the option to de-select the rune sockets, but this is not ideal. I would prefer the behavior be to add rune sockets instead of assuming all items have them.

Additionally, if the rune sockets are not removed prior to "Search"ing, then the item will have the rune sockets regardless. 

TLDR; Can we update the socket behavior to not assume all items have sockets?

**Comment by Kvan7** [comment]
#223
